cobra/core/model.py

```diff
@@ -149,6 +154,18 @@ def solver(self, value):
             return
         self._solver = interface.Model.clone(self._solver)
 
+    @property
+    def tolerance(self):
+        return self._tolerance
+
+    @tolerance.setter
+    def tolerance(self, value):
+        solver_config = self._solver.configuration
+        solver_config.tolerances.feasibility = value
```

Comment:
Some of these will raise `AttributeError` if they are not implemented on the specific solver. This could be more nicely handled on optlang but that's where it is at the moment. So each line individually should probably be wrapped in a `try` ... `except`.

Reply:
Ah it completely slipped my mind. Will fix that.